backend/database_handler/transactions_processor.py

```
@@ -562,3 +572,25 @@ def get_previous_transaction(
             if closest_transaction
             else None
         )
+
+    def set_transaction_timestamp_last_vote(self, transaction_hash: str):
+        transaction = (
+            self.session.query(Transactions).filter_by(hash=transaction_hash).one()
+        )
+        transaction.last_vote_timestamp = int(time.time())
+        self.session.commit()
+
+    def increase_transaction_rotation_count(self, transaction_hash: str):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)